install/docker/forseti

```diff
@@ -1,44 +0,0 @@
-# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
Is there any impact from deleting these `/install/docker` files?  Do we know if any users are using it?  Any documentations need to be updated?

Reply:
Am unaware of customer impact.  The install/scripts/docker_*.sh scripts have been updated to mitigate impact.  The documentation will be updated.

Reply:
I can put those files back then make a note in the docs that they will be deprecated by Forseti 2.x.

What do you think?

Reply:
+1  Yes, I think that having a transition period with clear note + time for deprecation is a much safer approach.